test/reservation_test.rb

```diff
@@ -0,0 +1,105 @@
+require_relative "test_helper"
+
+describe Hotel::Reservation do
+  before do
+  end
+
+    describe "initialize" do
+      it "determines if an invalid range is given" do
+      @end_date = "2020-08-04"
+      @start_date = "2020-08-06"
+
+      new_reservation = Hotel::Reservation.new(@start_date, @end_date, nil)
```

Comment:
This test produces an error because this line appropriately raises an error:
```suggestion
      new_reservation = Hotel::Reservation.new(@start_date, @end_date, nil)
```
This line needs to be in the expect{} so that the test doesn't itself raise an error.

```suggestion
      expect{new_reservation = Hotel::Reservation.new(@start_date, @end_date, nil)}.must_raise StandardError
```